### PR TITLE
[LIVY-389] [Release] 0.4.0-incubating release update

### PR DESCRIPTION
**Pull request by ajbozarth**
Website updates for 0.4.0-incubating, including download page, history page and 0.4.0-incubating release post.

### CODE REVIEW

**Comment by ajbozarth** [comment]
Updated the JIRA and apache repo links. Only remaining update upon release should be changing to the correct release date.

**Comment by ajbozarth** [comment]
Updated date to ~~8/30 (assuming thats correct)~~ 9/1 and removed the WIP

**Comment by jerryshao** [comment]
Thanks @ajbozarth for your update, let me merge it.

File: site/download.md [hunk 1]
```diff
@@ -30,40 +30,49 @@ limitations under the License.
...
+            <th><b>Archive</b></th>
+            <th><b>MD5</b></th>
+            <th><b>SHA-512</b></th>
+            <th><b>signature</b></th>
+        </tr>
+    </thead>
+    <tbody>
+        <tr>
+            <td>{{ site.data.project.name }} {{site.data.project.latest_release}} (tar.gz)</td>
+            <td><a href="http://www.apache.org/dyn/closer.lua/incubator/{{site.data.project.unix_name}}/{{site.data.project.latest_release}}/{{site.data.project.unix_name}}-{{site.data.project.latest_release}}-bin.zip">tar.gz</a></td>
```
Comment (jerryshao) [inline site/download.md:46]: @ajbozarth here actually should be zip, not tgz, can you please file a PR to fix it?

**Pull request closed without merging**